GX-TimeFrequency/tests/test_minimax/output_32_freq_time_points.dat

```diff
@@ -0,0 +1,34 @@
+ Minimax Grid
```

Comment:
Actually, this shouldn't be added either - only the reference should exist, not test outputs (You or I can look at adding to the gitignore to avoid this)

Reply:
- [ ] Remove